pkg/cmd/flags/capture.go

```diff
@@ -104,7 +105,10 @@ func PrepareCapture(captureSlice []string) (tracee.CaptureConfig, error) {
 
 	capture.OutputPath = filepath.Join(outDir, "out")
 	if clearDir {
-		os.RemoveAll(capture.OutputPath)
+		err := os.RemoveAll(capture.OutputPath)
+		if err != nil {
+			logger.Error("Removing all", "path", "error", err)
```

Comment:
This might not be a real problem if the files failed to be removed are not colliding with capture ones.
I think it should be a Warn level.
If the capture open file fails it should be a real error.